src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -60,6 +65,11 @@ export default App.extend({
       'cancel:comment': this.onCancelNewComment,
     });
   },
+  showAttachments() {
+    if (!this.action.hasAttachments()) return;
```

Comment:
Do we want to add an additional check here for `this.attachmentsCollection.length`? Just remembered that you mentioned this previously.

Is it possible `this.action.hasAttachments()` returns `false` (meaning `action.relationships._files` is empty), but the `/actions/{actionId}/relationships/files` api request returns files for the action? 

In that scenario, an empty `Attachments` section would be shown in the sidebar.

Reply:
Or I guess we could replace `this.action.hasAttachments()` with `this.attachmentsCollection.length` as well.

Reply:
just checking the collection I think makes more sense and will be more correct, even if the attachment icon doesn't get updated in the list